benchmark/1mo_benchmark.yml

```diff
@@ -1,4 +1,4 @@
----
+&---
```

Comment:
What does `&` do here?

Reply:
Same comment as above. This was fixed in my local branch but not pushed.